emsdk.py

```diff
@@ -2182,6 +2182,17 @@ def load_releases_tags():
   if extra_release_tag:
     tags.append(extra_release_tag)
 
+  # Adds the currently installed SDK version, which might be custom
+  # version.  This means it will show up in `list` and work with
+  # `construct_env`
```

Comment:
is this function the right place to do this? I think `load_releases_tags` should just load the tags from the network. this seems like a separate operation?

Reply:
load_releases_tags is mostly loading tags from a local file: emscripten-releases-tags.txt.  Its nothing to do with the network.

Reply:
I guess we could rename this function "find_available_sdk_releases" ?

Reply:
It can have extra versions inject either from the command line (`emsdk install sdk-upstream-<SHA>`) or from a version that already was installed previously using that method.

Reply:
I see what you mean, yeah.